mscore/mixer.cpp

```diff
@@ -125,6 +126,32 @@ Mixer::Mixer(QWidget* parent)
       retranslate(true);
       }
 
+//---------------------------------------------------------
+//   toggleMuteTracks
+//---------------------------------------------------------
+
+void Mixer::toggleMuteTracks()
+      {
+      for (int i =0; i < trackList.length(); i++)
+            {
+            trackList.at(i)->mti()->toggleMute();
+            }
+      //make sure expanded tracks are toggled as well
+      QSetIterator<Part *> pi(expandedParts);
+
+      while (pi.hasNext())
+            {
+            Part *p = pi.next();
+            for (Channel *chan: p->instrument()->channel()) {
+                  if(chan->mute()){
```

Comment:
Isn't it better to write `chan->setMute(!chan->mute());`? Without the `if` statement.